cms/jinja2/templates/base.html

```diff
@@ -8,72 +8,76 @@
 {% endset %}
 
 {# fmt:off #}
```

Comment:
as per question at S&T, we need to pass hreflangs to pageConfig. https://service-manual.ons.gov.uk/design-system/foundations/base-page-template#hreflangs